app/src/main/java/com/github/sdpteam15/polyevents/helper/GoogleMapHelper.kt

```diff
@@ -33,25 +37,30 @@ data class IconAnchor(var anchorWidth: Float, var anchorHeight: Float)
 @SuppressLint("StaticFieldLeak")
 object GoogleMapHelper {
     var map: MapsInterface? = null
-    var uid = 0
+    var uidArea = 0
+    var uidZone = 0
```

Comment:
Okay for now as we're working in local but using firebase we will need to remove this id  and take the one generated by firebase